docs/source/interfaces/Engineering Diffraction Test Guide.rst

```diff
@@ -78,8 +80,8 @@ If the data is captured in event mode (i.e.. Event workspaces) it will first
 need :ref:`preProcessing-Engineering_Diffraction_test-ref` .
 
 A .nxs file containing histogram data needs to be selected for example *ENGINX00193749.nxs*
-this can have all banks summed, specific spectra and list or use a detector grouping to create
-a texture. 
+/ 305761 on archive this can have all banks summed, specific spectra and list or use a
```

Comment:
```suggestion
/ 305761 on archive. This can have all banks summed, specific spectra and list or use a
```
I think this might need some form of punctuation here